public/app/features/panel/panel_menu.js

```diff
@@ -14,6 +14,7 @@ function (angular, $, _, Tether) {
           '<span class="panel-title drag-handle pointer">' +
             '<span class="icon-gf panel-alert-icon"></span>' +
             '<span class="panel-title-text drag-handle">{{ctrl.panel.title | interpolateTemplateVars:this}}</span>' +
+            '<span class="panel-help-text"><info-popover mode="bold">{{ctrl.panel.helpText}}</info-popover></span>' +
```

Comment:
Thanks! :) 

But there is an issues I see with the tooltip sass and we need to fix it before merging this PR to the master.

If help text on the cannot fit in the browser window on dashboard launch, then either it does not show up or show up somewhere else on the dashboard.

To reproduce:

1. Add 5 rows on a dashboard, make sure the 5th row can only be seen when you scroll down on the dashboard.
2. Add help text to any panel on the 5th row.
3. Hover over the info-circle, you should see that it will appear somewhere else on the dashboard.
4. But when you zoom out your dashboard in the browser and can see the 5th row without scrolling down, the help text will appear correctly.